tests/conftest.py

```diff
@@ -265,41 +258,71 @@ def i13_179623():
     return "tests/test_data/raw_data/i13/360/179623.nxs"
 
 
+@pytest.fixture
+def i12_119647():
+    # 1.8k projections, 20gb dataset, 180 degrees scan
+    return "tests/test_data/raw_data/i12/119647.nxs"
+
+
 ############## --Ground Truth references-- #################
 
 
 @pytest.fixture
-def gpu_diad_FBP_k11_38731_npz():
+def FBP3d_tomobar_k11_38731_npz():
     # 10 slices numpy array
-    return np.load("tests/test_data/raw_data/diad/gpu_diad_FBP_k11-38731.npz")
+    return np.load("tests/test_data/raw_data/diad/FBP3d_tomobar_k11-38731.npz")
 
 
 @pytest.fixture
-def gpu_diad_FBP_k11_38730_npz():
+def FBP3d_tomobar_k11_38730_npz():
     # 10 slices numpy array
-    return np.load("tests/test_data/raw_data/diad/gpu_diad_FBP_k11-38730.npz")
+    return np.load("tests/test_data/raw_data/diad/FBP3d_tomobar_k11-38730.npz")
 
 
 @pytest.fixture
-def gpu_FBP_TVdenoising_i13_177906_npz():
+def FBP3d_tomobar_i12_119647_npz():
     # 10 slices numpy array
-    return np.load("tests/test_data/raw_data/i13/gpu_FBP_TVdenoising_i13_177906.npz")
+    return np.load("tests/test_data/raw_data/i12/FBP3d_tomobar_i12_119647.npz")
 
 
 @pytest.fixture
-def gpu_FBP_paganin_i13_179623_npz():
+def FBP2d_astra_i12_119647_npz():
     # 10 slices numpy array
-    return np.load("tests/test_data/raw_data/i13/360/gpu_FBP_paganin_i13_179623.npz")
+    return np.load("tests/test_data/raw_data/i12/FBP2d_astra_i12_119647.npz")
+
+
+# @pytest.fixture
+# def gpu_FBP3d_i13_177906_npz():
+#     # 10 slices numpy array
+#     return np.load("tests/test_data/raw_data/i13/gpu_FBP3d_i13_177906.npz")
```

Comment:
Is there going to be more work done in this PR to allow this to be uncommented, or is this intended to be commented-out?

Reply:
There was a test using this dataset, but I added a different dataset instead. So I'll remove the commented test probably. thx